lib/internal/validators.js

```diff
@@ -163,13 +178,16 @@ function validateSignalName(signal, name = 'signal') {
   }
 }
 
-const validateBuffer = hideStackFrames((buffer, name = 'buffer') => {
-  if (!isArrayBufferView(buffer)) {
-    throw new ERR_INVALID_ARG_TYPE(name,
-                                   ['Buffer', 'TypedArray', 'DataView'],
-                                   buffer);
-  }
-});
+const validateBuffer = hideStackFrames(
```

Comment:
can these changes be seperate? some of the line wrapping changes look cosmetic, btw.

Reply:
Yes, with the caveat that several of these changes are only used by the QUIC code.